pubpub/pubpub
# feature: Submissions tab

This PR adds a Submissions tab to the Collection scope of the dashboard, hidden behind the `submissions` feature flag. It also has a first pass at a `SubmissionWorkflowEditor`. Follow-on work for this PR will include:

- Querying and sending real `SubmissionWorkflow` model from #1617
- Creating a new `SubmissionWorkflowEditor` wrapper that persists a real model via the API
- Switching between viewing submitted Pubs and editing an existing workflow

Because there is a lot of integration work left to do, there are a lot of dead ends (and odds and ends) in this PR that I will flag with my own review comments.

_Test plan:_
- Enroll yourself in the `submissions` feature flag (or just log in as PubPub Team)
- Navigate to a Collection in the dashboard and verify that you can see the Submissions tab
- Make sure you can use all the workflow editor controls, and that your WIP persists via `localStorage` when refreshing
- Click "Save and continue" and verify that you are taken to a mostly-blank dashboard

_Screenshot:_
<img width="1904" alt="image" src="https://user-images.githubusercontent.com/2208769/140995155-d30cd3ac-3369-4b47-98a3-b4e3a790008d.png">

Resolves #1613
Resolves #1614
